components/DropdownMenu/dropdownMenu.module.scss

```diff
@@ -1,6 +1,6 @@
 @use 'sass:color';
-@use '_variables.module.scss';
-@import 'colors.module.scss';
+@use '../../scss/variables.module.scss';
```

Comment:
Info: you can omit the leading underscore when importing a Sass partial. ([Docs](https://sass-lang.com/documentation/at-rules/use#partials))